sma-webconnect.html

```diff
@@ -47,6 +47,7 @@
   <label for="node-input-device_selection"><i class="fa fa-gears"></i> Device</label>
   <select id="node-input-device_selection">
     <option selected value="sb_tripower">SB TRIPOWER 8.0/10.0</option>
+    <option value="sb">SB 3.0/3.6/4.0/5.0 (1AV-40)</option>
```

Comment:
Suggestion: change `sb` to `sb_inverter`

Reply:
I'd be okay with the naming here, as the official naming is:

Sunny Boy
Sunny Boy Storage
Sunny Tripower (that's currently sb_tripower, so the 'b' is mistaken)

But I agree in general that the three const we're using here should better be namespaced or so. But as long as we only have those three it's okay. I will keep it in mind for a refactoring.

Reply:
Then maybe we should start with this unit by using `sunny_boy` as name. I have prepared an update in which the presets are stored in separate files where the file name also serves to access the presets. After this is merged i would then just move the new preset to a separate file.